spec/feature_flagger/notifier_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Notifier do
+    let(:feature_key)           { 'account:email_marketing:whitelabel' }
+    let(:legacy_feature_key)   { 'account' }
+    let(:resource_id)   { 'resource_id' }
+    let(:resource_name) { 'account' }
+
+    describe '#send' do
+      context 'When having a callback configured' do
+        let(:notifier_callback) { spy(lambda { |event| }, :is_a? => Proc)}
+        let(:notifier) { Notifier.new(notifier_callback)}
+        let(:generic_event) {
+          {
+            type: FeatureFlagger::Notifier::RELEASE,
+            model: resource_name,
+            feature: feature_key,
+            id: resource_id
+          }
+        }
+        context 'When call the lambda function' do
```

Comment:
Which cenario this is covering? It's not clear to me.

Reply:
Good question, this spec is unnecessary, we have the same test case covered in the next context.

```ruby
          context 'When do a release' do
            it { expect(notifier_callback).to have_received(:call).with(generic_event) }
          end

```